pkg/update/update.go

```diff
@@ -101,3 +101,29 @@ func TrimVersionInfo(pkg string) (string, string) {
 
 	return name, version
 }
+
+// ComparePackages compares the devVersions and the runtimeVersions
+func ComparePackages(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+
+	counts := make(map[string]int)
```

Comment:
Another simpler way to accomplish the same thing would be to- 
* Loop over slice `a` to create a `map[string]bool`. Key will be entire package i.e `pkg@~v1.0.0`, etc. 
* Loop over the slice `b` and check if the element is present in the map. If the element isn't found, return `false`. 

It's similar but we do the same with 2 for loops instead of 3 by utilising the map. Does that make sense?